
<html>
<head>
<title>Workshop on Bio-Design for Portability (BD4P)</title>
</head>

<body>
<font size="6"><b>
Workshop on Bio-Design for Portability (BD4P)</b></font><br>
<br>
<font size="4">
8 July 2019<br><br>

Room FW11<br>
<a href="https://www.cst.cam.ac.uk"target="_blank">Department of Computer Science and Technology</a><br>

University of Cambridge, UK (co-located with <a href="http://www.iwbdaconf.org/2019/"target="_blank">IWBDA 2019</a>) <br>

<br>
<hr width="200" align="LEFT">
<br>
The workshop is sponsored by the <a href="https://portabolomics.ico2s.org/"target="_blank">EPSRC Portabolomics project</a>.<br>
<br>
There is no registration fee, but for catering purposes <b>please register</b> <a href="https://bit.ly/2Nj9bgR"target="_blank">here</a>. <br>
<br>
<hr width="200" align="LEFT">
<p>
</p>

<b>Programme</b>
<ul>
  <li> 1:30-1:40pm &ensp;  <a href="https://pzuliani.github.io"target="_blank">Paolo Zuliani</a> (Newcastle): Welcome<br><br>
  <li> 1:40-2:20pm &ensp; <a href="https://biocomputelab.github.io/team/thomas-gorochowski/"target="_blank">Thomas Gorochowski</a> (Bristol):
                    <em>Towards a complete and quantitative view of genetic circuit function</em><br><br>
  <li> 2:20-3:00pm &ensp; <a href="https://jakebeal.github.io/"target="_blank">Jacob Beal</a> (BBN Technologies)<br><br>
  <li> 3:00-3:30pm &ensp; <em>Break</em><br><br>
  <li> 3:30-4:00pm &ensp; <a href="https://portabolomics.ico2s.org/dr-wendy-smith/"target="_blank">Wendy Smith</a> (Newcastle)<br><br>
  <li> 4:00-4:30pm &ensp; <a href="https://sites.google.com/view/shouyongjiang/"target="_blank">Shouyong Jiang</a> (Lincoln):
                    <em>Game-theoretic approaches for in silico strain design in metabolic engineering</em><br><br>
  <li> 4:30-5:00pm &ensp; <a href="http://www.shirt-ediss.me/"target="_blank">Benjamin Shirt-Ediss</a> (Newcastle):
                    <em>Heuristic Sequence Selection for Nucleic Acid Origami Scaffolds</em><br><br>
  <li> 5:00-5:30pm &ensp; Discussion<br><br>
</ul>

<hr width="200" align="LEFT">

<b>Abstracts</b>
<br>
<br>
<b>Towards a complete and quantitative view of genetic circuit function</b>
<br>
Thomas E. Gorochowski
<br>
Royal Society University Research Fellow
<br>
School of Biological Sciences, University of Bristol, UK
<br>
<br>

Synthetic genetic circuits are composed of many interconnected parts that must function together in concert to implement desired biological computations. A major challenge when developing new circuits is that genetic parts often display unexpected changes in their performance when used in new ways. These arise due to contextual effects or unintended interactions with the host cell. In this talk, I will demonstrate how we have been using sequencing technologies and mathematical modelling to tackle this problem. First, I will show how RNA-sequencing can be used to measure the function of every transcriptional part making up large genetic circuits. This enables us to better understand why some designs fail and helps pin point the root cause. Then, I will present some recent work where we combined RNA-sequencing with ribosome profiling and RNA spike-in standards to enable the first large-scale characterisation of transcriptional and translational parts in absolute units. Such capabilities provide a more complete and quantitative view of the inner workings of genetic circuits and improve our understanding of the rules governing the effective construction of larger and more complex biological systems


<p>
</p>
<hr width="100" align="LEFT">
<br>
<b>Game-theoretic approaches for in silico strain design in metabolic engineering</b>
<br>
Shouyong Jiang
<br>
Lecturer
<br>
College of Science, University of Lincoln, UK
<br>
<br>
The computational reconstruction of genome-scale metabolic networks for microorganisms enables not only improved understanding of metablism but also valuable guidance in metabolic engineering. Genome-scale metabolic network analysis can provide useful prediction of candidate targets to improve biochemical production, yield, and titer.  Particularly,  the tradeoff between cell growth and biochemical production is very important in identifying genetic manipulations. We consider this mutally competitive relationship from a game-theoretic point of view, namely, one game player is the host cell and the other is the product one wants to overproduce. We demonstrate our approach through E. coli as a production platform to produce a number of biochemicals. Results show that the game-theoretic approach turns out to be very useful in predicting novel strain design strategies for either single or multiple products.

<p>
</p>
<hr width="100" align="LEFT">
<br>

<b>Heuristic Sequence Selection for Nucleic Acid Origami Scaffolds</b>
<br>
<br>
Ben Shirt-Ediss
<br>
Research Associate
<br>
Interdisciplinary Computing and Complex Biosystems Research Group
<br>
School of Computing, Newcastle University, Newcastle-upon-Tyne, UK
<br>
<br>
(joint work with Emanuela Torelli and Natalio Krasnogor)
<br>

<p>
Nucleic acid origami exploits predictable molecular self-assembly as a means to engineer nano-sized functional objects from DNA and RNA. Now well-proven in vitro, nucleic acid origami is finding increasing applications in next generation medicine [1].
</p>
<p>
Computational design methods for nucleic acid origami have tended to exclusively focus on the problem of geometric design - that is, how to line up staple crossovers such that the final nanoshape has the intended shape, free from unwanted twists or bends [2, 3, 4]. However, in practice, all origami nanoshapes self-assemble into their final geometry from monomer single strands, and success of this kinetic process is also heavily dependent on the nucleic acid sequences used. The equally important sequence design problem - that is, how to choose a scaffold sequence that folds in high yield - has received relatively little attention as compared to the geometric design problem. This may be explained by the fact that, traditionally, the only choices for origami scaffolds were fixed natural biological sequences (e.g. the M13 virus genome). However, decreasing costs of nucleic acid polymer synthesis has enabled the production of custom synthetic scaffolds, enabling scaffold sequence to also count as an important design parameter in the production of origami nano-objects. Initial works relevant to the sequence design problem have been the development of qualitative sequence-dependent origami folding models [5, 6] and basic heuristic approaches to scaffold sequence design [7, 8].
</p>
<p>
In this work, we present a new improved heuristic approach for selecting scaffold sequences that are likely to fold into a target DNA or RNA origami object (2D or 3D) in high yield.  Given a large initial pool of origami scaffold sequences, our multi-objective design algorithm OnePot applies five scoring criteria to calculate which subset of scaffold sequences have the least number of barriers preventing them from folding into the target nanostructure (in a single pot reaction). Further, the initial scaffold sequence pool may be initialised to exclude biologically relevant sub-sequences, making our method applicable for designing origami destined for in-cell (and possibly cross-species) applications.
</p>

<ol>
 <li> 	Chandrasekaran, A. R. Designer DNA Architectures: Applications in Nanomedicine. Nanobiomedicine 2016, 3, 6.
 <li> 	Douglas, S. M.; Marblestone, A. H.; Teerapittayanon, S.; Vazquez, A.; Church, G. M.; Shih, W. M. Rapid Prototyping of 3D DNA-Origami Shapes with CaDNAno. Nucleic Acids Res. 2009, 37 (15), 5001–5006.
 <li>  	Jun, H.; Zhang, F.; Shepherd, T.; Ratanalert, S.; Qi, X.; Yan, H.; Bathe, M. Autonomously Designed Free-Form 2D DNA Origami. Sci. Adv. 2019, 5 (1), eaav0655.
 <li> 	Veneziano, R.; Ratanalert, S.; Zhang, K.; Zhang, F.; Yan, H.; Chiu, W.; Bathe, M. Designer Nanoscale DNA Assemblies Programmed from the Top Down. Science (80-. ). 2016, 352 (6293), 1534–1534.
 <li> 	Arbona, J.-M.; Aimé, J.-P.; Elezgaray, J. Cooperativity in the Annealing of DNA Origamis. J. Chem. Phys. 2013, 138 (1), 015105.
 <li> 	Dannenberg, F.; Dunn, K. E.; Bath, J.; Kwiatkowska, M.; Turberfield, A. J.; Ouldridge, T. E. Modelling DNA Origami Self-Assembly at the Domain Level. J. Chem. Phys. 2015, 143 (16), 165102.
 <li> 	Kozyra, J.; Ceccarelli, A.; Torelli, E.; Lopiccolo, A.; Gu, J.-Y.; Fellermann, H.; Stimming, U.; Krasnogor, N. Designing Uniquely Addressable Bio-Orthogonal Synthetic Scaffolds for DNA and RNA Origami. ACS Synth. Biol. 2017, 6 (7), 1140–1149.
 <li> 	Engelhardt, F. A. S.; Praetorius, F.; Wachauf, C. H.; Brüggenthies, G.; Kohler, F.; Kick, B.; Kadletz, K. L.; Pham, P. N.; Behler, K. L.; Gerling, T.; et al. Custom-Size, Functional, and Durable DNA Origami with Design-Specific Scaffolds. ACS Nano 2019, 13 (5), 5015–5027.
</ol>

</font>


</body>
</html>
